src/EnergyPlus/DataAirLoop.hh

```diff
@@ -85,6 +85,9 @@ namespace DataAirLoop {
         Array1D_int TermUnitCoolSizingIndex;       // Air terminal sizing numbers for zones cooled by this air loop
         Array1D_int TermUnitHeatSizingIndex;       // Air terminal sizing numbers for zones heated by this air loop
         Array1D<HVAC::AirDuctType> SupplyDuctType; // 1=main, 2=cooling, 3=heating, 4=other
+        Array1D_int SupplyDuctBranchNum;           // Supply duct branch number
+        Array1D_int SupplyAirPathNum;              // Supply air path indexes
+        Array1D_int ReturnAirPathNum;              // Return air path indexes
```

Comment:
OK, but I still think EPVector is a better option for a lightweight 1-based array.  Both Array1D and EPVector are destined to become regular old C++ vectors, so it's not crucial, but @jasondegraw and company have set up EPVector as a great piece of that transition.  

Anyway, no huge deal, carrying on...

Reply:
Using EPVector is a good idea here.